src/Microsoft.Health.Fhir.Core/Features/Operations/Export/ExportJobTask.cs

```diff
@@ -27,6 +28,8 @@ namespace Microsoft.Health.Fhir.Core.Features.Operations.Export
 {
     public class ExportJobTask : IExportJobTask
     {
+        private static readonly Regex Base64FormatRegex = new Regex("^([A-Za-z0-9+/]{4})*([A-Za-z0-9+/]{3}=|[A-Za-z0-9+/]{2}==)?$", RegexOptions.Compiled | RegexOptions.Singleline);
```

Comment:
@feordin since this is still behind a feature flag, do you think this check is needed?

Reply:
We do not need to check any previously created continuation tokens.  The reindex functionality is not E2E functional yet (but soon!) so any created jobs would need to be re-created and run anyway.  However, We may need to add logic to encode any returned continuation tokens which come back in the search results if the search service expects them to be encoded.  I think we can simply allow this update to go through, and then test a reindex job with continuation tokens to be sure.